README.md

```diff
@@ -48,7 +48,7 @@ pushed to your docker repository.
 Use the appropriate make targets or scripts in `hack`:
 
 - `make dev`: Deploys the serverless-operator without deploying Knative Serving, Eventing and Kafka components.
-- `make install`: Scales the cluster appropriately, deploys serverless-operator, Knative Serving and Eventing.
+- `make install`: Scales the cluster appropriately, deploys serverless-operator, Knative Serving, Eventing and Kafka components.
```

Comment:
This is not accurate. Probably happened during rebases (I've been there)

Reply:
IMO, changes in this file should be completely reverted.

Reply:
sure